src/models/issues.d.ts

```diff
@@ -0,0 +1,82 @@
+interface User {
+  login: string;
+  id: number;
+  node_id: string;
+  avatar_url: string;
+  gravatar_id: string;
+  url: string;
+  html_url: string;
+  followers_url: string;
+  following_url: string;
+  gists_url: string;
+  starred_url: string;
+  subscriptions_url: string;
+  organizations_url: string;
+  repos_url: string;
+  events_url: string;
+  received_events_url: string;
+  type: string;
+  site_admin: boolean;
+}
+
+interface Reaction {
+  url: string;
+  total_count: number;
+  laugh: number;
+  hooray: number;
+  confused: number;
+  heart: number;
+  rocket: number;
+  eyes: number;
+}
+
+interface Assignee extends User {}
```

Comment:
I see that you want to re-use the User interface's properties in Assignee by extending, but you aren't adding any additional properties to Assignee. Consider only having User then in IIssue, you can have ` assignee: User | null; assignees : User[] `

Reply:
Thank you, it's been fixed